### DISCUSSION TITLE
How to track split testing on Netlify

### DISCUSSION THREAD

**Discussion by ghk-arolla** [plausible/analytics#3058] [Anything Else]
Netlify allows us to do split testing (https://docs.netlify.com/site-deploys/split-testing/)

That is, to deploy a "branch" in addition to "main" and they serve 50/50% each branch. 

So I need to see what the impact is on clicks or goals for each branch tested.

In hugo I can get the branch served, so info is available at the web code level.

But I can't find a simple way of tracking this in plausible.

Any advice or examples?

Nota : I read https://github.com/plausible/analytics/issues/134 (and I also am @divinerites on an other account)

**Comment by metmarkosaric** [comment]
hi @ghk-arolla! perhaps our custom properties for pageviews are useful for this? allows you to send custom data alongside each pageview which then allows you to segment the audience in your dashboard too. see https://plausible.io/docs/custom-pageview-props

**Comment by ghk-arolla** [comment]
Ahhhhh ..... genius !!!!

This will do the trick. I will also update my [plausible-hugo](https://github.com/plausible/analytics) plug-in with this option.

Thnaks marko.

Divinerites.

**Comment by ghk-arolla** [comment]
An other question.

You says *"Custom properties is an upcoming premium feature that's free-to-use during the private preview. Pricing will be announced soon."*

I can't find anywhere information about "premium feature". Is that new ? Is it different from our paid account ?

**Reply by metmarkosaric** [reply]
yeah, the "premium" is an upcoming thing so we haven't announced details yet. but here's more for you:

as GA is about to shut down we've put a lot of time into building 3 top requests from ecommerce and other businesses looking to move away from GA (custom properties, monetary values and funnels). these will become a part of our "business" tier later this year which is different from a regular tier. we've decided not to change the prices of our regular plans but to allow those people who are interested in business features to upgrade to get those features instead

we plan to run these three features as a free to use preview for 3 months starting from july 1st (the date GA stops counting stats) during which we will announce the full details of the premium tier and allow you to upgrade in your account settings if you wish to do so. custom properties are already available to be tested as we finished those first and the other two are coming over the next couple of weeks

**Comment by ghk-arolla** [comment]
OK Thanks.

But i have to pass the testing 3 months as I'll not have the money to upgrade to business prices.
I'll have to think for an other solution if it is feasible.

Thanks for the clear response anyway.
